auto_submit/lib/action/git_cli_revert_method.dart

```diff
@@ -67,15 +67,19 @@ class GitCliRevertMethod implements RevertMethod {
       retryIf: (Exception e) => e is NotFoundException,
     );
 
-    log.info('found branch ${slug.fullName}/${branch!.name}, safe to create revert request of ${pullRequest.number!}.');
+    log.info('found branch ${slug.fullName}/${branch!.name}, safe to create revert request of ${pullRequestToRevert.number!}.');
+
+    final Set<String> prToRevertReviewers = await getOriginalPrReviewers(githubService, slug, pullRequestToRevert.number!);
 
     final RevertIssueBodyFormatter formatter = RevertIssueBodyFormatter(
       slug: slug,
-      originalPrNumber: pullRequest.number!,
+      originalPrNumber: pullRequestToRevert.number!,
       initiatingAuthor: initiatingAuthor,
       revertReason: reasonForRevert,
-      originalPrTitle: pullRequest.title,
-      originalPrBody: pullRequest.body,
+      originalPrAuthor: pullRequestToRevert.user!.login,
+      originalPrReviewers: prToRevertReviewers,
```

Comment:
Nit: good to have consistent variable names: `pullRequestToRevert`+`pullRequestToRevertReviewers` or `prToRevert`+`prToRevertReviewers`.

Reply:
Gotcha. I will clean this up for clarity.